src/api/cdc/openapi.io.yaml.tpl

```
@@ -0,0 +1,178 @@
+openapi: 3.0.0
+info:
+  title: Carta della Cultutra REST APIs
+  description: API to request Carta della Cultura
+  version: 0.0.1
+servers:
+- url: ${host}
+
+paths:
+  /beneficiario/stato:
+    get:
+      summary: Lettura stato beneficiario per ogni anno
+      operationId: getStatoBeneficiario
+      responses:
+        "200":
+          description: Stato Beneficiario per Anno
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ListaStatoPerAnno'
+        "404": 
+          description: Risorsa non trovata
+        "401":
+          description: Utente non autorizzato
+        "403": 
+          description: Utente non loggato
+        "500":
+          description: Errore interno.
+      # JWT Token (RSA+256)
+      security:
+        - BearerAuth: []
+
+  /beneficiario/registrazione:
+    post:
+      tags:
+      - /secured/utente
+      summary: 'Richiesta Carta della Cultura da parte del beneficiario'
+      operationId: registraBeneficiario
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AnniRiferimento'
+      responses:
+        "200":
+          description: Richiesta OK per almeno un anno
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ListaEsitoRichiestaPerAnno'
+        "400":
+          description: Validazioni non superate
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RichiestaCartaErrata'
+        "404": 
+          description: Risorsa non esiste
+        "401":
+          description: Utente non autorizzato
+        "403":
+          description: Utente non loggato
+        "500":
+          description: Errore interno.
+      # JWT Token (RSA+256)
+      security:
+        - BearerAuth: []
+  
+
+components:
+  securitySchemes:
+    BearerAuth:
+      type: http
+      scheme: bearer
+  schemas:
+    StatoBeneficiario:
+      type: string
+      enum: 
+        - ATTVABILE
+        - INATTIVABILE 
+        - ATTIVO # carta richiesta e risposta ok da INPS
+        - VALUTAZIONE # carta richiesta ma non si conosce la risposta da INPS
+        - INATTIVO # carta richiesta e risposta negativa da INPS
+    EsitoRichiesta:
+      type: string
+      enum: 
+        - INIZIATIVA_TERMINATA
+        - ANNO_NON_AMMISSIBILE
+        - CIT_REGISTRATO
+    Anno: 
+      type: string 
+      pattern: '^\d{4}$'
```

Can you add a double backslash?
```suggestion
      pattern: '^\\d{4}$'
```
Unfortunately on the app we are using an old version of `openapi-codegen-ts` and we suffer from this problem.